src/Listener/RestExceptionListener.php

```diff
@@ -55,7 +56,7 @@ public function onKernelException($event)
             return;
         }
 
-        $exception = $event->getException();
+        $exception = $event->getThrowable();
```

Comment:
No such method in \Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent of symfony 3.4, see https://github.com/symfony/symfony/blob/3.4/src/Symfony/Component/HttpKernel/Event/GetResponseForExceptionEvent.php

Reply:
Right, added BC support for a version that stopped it's support on 2021-11-30. :+1: